src/org/labkey/test/components/glassLibrary/components/RbtCombo.java

```diff
@@ -0,0 +1,133 @@
+package org.labkey.test.components.glassLibrary.components;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.selenium.EphemeralWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class RbtCombo extends WebDriverComponent<RbtCombo.ElementCache>
```

Comment:
I don't think this is a LabKey developed component. Also, this name doesn't help people find it or know what it is for. Looking at the product code, I think this is a [`react-bootstrap-typeahead`](http://ericgio.github.io/react-bootstrap-typeahead/) component.
Move/rename this to `org.labkey.test.components.react.Typeahead` and add a class javadoc indicating what it represents. (Here's where the component is imported in the run builder: https://github.com/LabKey/provenance/blob/0e9c1e3e8046e16eb600bc8079dc160ff2238b6b/src/client/RunBuilder/PropertyLookup.tsx#L3)

@labkey-martyp Can you confirm?